src/gui.js

```diff
@@ -5392,7 +5414,7 @@ IDE_Morph.prototype.openProject = function (name) {
         this.setProjectName(name);
         str = localStorage['-snap-project-' + name];
         this.openProjectString(str);
-        this.setURL('#open:' + str);
+        this.setURL('open:' + str);
```

Comment:
Slightly unnecessary, but so is `#`.
We should consider removing this altogether.